### PR TITLE
[Cyberint] - updated mapper, classifier and layout

### PR DESCRIPTION
**Pull request by ShiratG**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Cyberint/ReleaseNotes/1_0_3.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+##### Cyberint (classifier)
+- Updated classifier according to new schema.
+
+#### Layouts
+##### Cyberint Incident Layout
+ - Updated incident and indicator layouts to content pack format.
+<!--
+##### layoutscontainer-Cyberint_Incident_Layout.json
+ - Updated incident and indicator layouts to content pack format.
+-->
```
Comment (reutshal) [inline Packs/Cyberint/ReleaseNotes/1_0_3.md:18]: Please remove if not needed

File: Packs/Cyberint/ReleaseNotes/1_0_3.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+#### Incident Types
+##### Cyberint Incident
+- Updated incident and indicator layouts to content pack format.
+
+#### Incident Fields
+##### incident_createddate
+- Added **Created Date** to Incident Fields
```
Comment (reutshal) [inline Packs/Cyberint/ReleaseNotes/1_0_3.md:26]: I'm not sure we should mention new entities in the RN. Does it fail without it?

File: Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-    "id": "incident_createdby",
+    "id": "indicator_cyrenmalwarefamily",
```
Comment (reutshal) [inline Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json:2]: Why are you changing the incident field's id?

**Review by reutshal — CHANGES_REQUESTED** [review]

File: Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json (hunk 3 above)
Comment (ShiratG) [reply Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json:2]: I was getting the following errors, so I tried to fix them: 

Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json: [ST102] - The file id has changed from indicator_cyrenmalwarefamily to incident_createdby

Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json: [ST103] - You've added fromversion to an existing file in the system, this is not allowed, please undo.

File: Packs/Cyberint/ReleaseNotes/1_0_3.md (hunk 2 above)
Comment (ShiratG) [reply Packs/Cyberint/ReleaseNotes/1_0_3.md]: Yes... 

Validating Packs/Cyberint/ReleaseNotes/1_0_3.md as releasenotes
id set validations for: Packs/Cyberint/ReleaseNotes/1_0_3.md
Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Target vector" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Cyberint Category" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Vulnerable CName Record" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "IOC" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Target Brand" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Source Category" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Impacts" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Related Entities" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "CyberintConfidence" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Recommandation" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Alert_Attachments" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Ticket_ID" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Expert Analysis" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Created Date" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the indicatorfield "Created By" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the mapper "Cyberint (mapper)" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the layoutscontainer "Cyberint Incident Layout" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "ClosureReason" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

**Pull request closed without merging**